### PR TITLE
Add TA list to mview command

### PR DESCRIPTION
**Pull request by bryanngzh**

### CODE REVIEW

File: src/main/java/seedu/address/model/person/UniqueTutorList.java [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                && internalList.equals(((UniqueTutorList) other).internalList));
+    }
+
+    @Override
+    public int hashCode() {
+        return internalList.hashCode();
+    }
+
+    /**
+     * Returns true if {@code persons} contains only unique persons.
```
Comment (leongdl135) [inline src/main/java/seedu/address/model/person/UniqueTutorList.java:138]: Should this be students instead?

File: src/main/java/seedu/address/ui/TutorListPanel.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ */
+public class TutorListPanel extends UiPart<Region> {
+    private static final String FXML = "TutorListPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(TutorListPanel.class);
+
+    @javafx.fxml.FXML
+    private ListView<Student> tutorListView;
+
+    /**
+     * Creates a {@code PersonListPanel} with the given {@code ObservableList}.
```
Comment (leongdl135) [inline src/main/java/seedu/address/ui/TutorListPanel.java:23]: Should this be TutorListPanel instead?

File: src/main/java/seedu/address/ui/TutorListPanel.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+     */
+    public TutorListPanel(ObservableList<Student> tutorList) {
+        super(FXML);
+        tutorListView.setItems(tutorList);
+        tutorListView.setCellFactory(listView -> new TutorListPanel.TutorListViewCell());
+    }
+
+    /**
+     * Custom {@code ListCell} that displays the graphics of a {@code Person} using a {@code PersonCard}.
```
Comment (leongdl135) [inline src/main/java/seedu/address/ui/TutorListPanel.java:32]: Should this be Student and TutorCard instead?

**Review by leongdl135 — COMMENTED** [review]
LGTM overall, just need to fix some minor Javadoc's issues.

File: src/main/java/seedu/address/model/person/UniqueTutorList.java (hunk 1 above)
Comment (bryanngzh) [reply src/main/java/seedu/address/model/person/UniqueTutorList.java]: Fixed, thanks! 👍

File: src/main/java/seedu/address/ui/TutorListPanel.java (hunk 2 above)
Comment (bryanngzh) [reply src/main/java/seedu/address/ui/TutorListPanel.java]: Fixed, thanks! 👍

File: src/main/java/seedu/address/ui/TutorListPanel.java (hunk 3 above)
Comment (bryanngzh) [reply src/main/java/seedu/address/ui/TutorListPanel.java]: Fixed, thanks! 👍

**Review by leongdl135 — APPROVED** [review]
LGTM

**Pull request merged**